docs/reference/glossary.md

```diff
@@ -442,244 +378,240 @@ A SpatialOS project includes (but isn't limited to):
 * Optional [snapshots](#snapshot) of the project’s [SpatialOS world](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/glossary#spatialos-world)
 * Configuration files, mostly containing settings for [deployments](#deploying) (for example, [launch configuration files](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/worker-configuration/launch-configuration))
 
-> Related:
+> **Related:**
 >
 > * [Project directory structure](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/reference/project-structure)
```

Comment:
link is broken

Reply:
https://docs.improbable.io/reference/13.8/shared/project-layout/files-and-directories